src/core/hle/service/hid/controllers/gesture.cpp

```diff
@@ -182,20 +395,32 @@ Controller_Gesture::GestureProperties Controller_Gesture::GetGestureProperties()
             static_cast<int>(active_fingers[id].x * Layout::ScreenUndocked::Width);
         gesture.points[id].y =
             static_cast<int>(active_fingers[id].y * Layout::ScreenUndocked::Height);
+
+        // Hack: There is no touch in docked but games still allow it
+        if (Settings::values.use_docked_mode.GetValue()) {
+            gesture.points[id].x =
+                static_cast<int>(active_fingers[id].x * Layout::ScreenDocked::Width);
+            gesture.points[id].y =
+                static_cast<int>(active_fingers[id].y * Layout::ScreenDocked::Height);
+        }
+
         gesture.mid_point.x += static_cast<int>(gesture.points[id].x / gesture.active_points);
         gesture.mid_point.y += static_cast<int>(gesture.points[id].y / gesture.active_points);
     }
 
     for (size_t id = 0; id < gesture.active_points; ++id) {
         const double distance =
```

Comment:
eliminates the need to re-cast distance to an f32
```suggestion
        const f32 distance =
```

Reply:
Also is distance meant to be a^2 + b^2 or sqrt(a^2 + b^2)

Reply:
I will have to confirm this one. Parameters did look like they are squared but now I'm in doubt

Reply:
It should be sqrt(a^2 + b^2)